src/NUnitFramework/framework/Internal/OSPlatform.cs

```diff
@@ -217,16 +249,13 @@ public OSPlatform(PlatformID platform, Version version)
         public OSPlatform(PlatformID platform, Version version, ProductType product)
             : this( platform, version )
         {
-            _product = product;
+            Product = product;
         }
 
         /// <summary>
         /// Get the platform ID of this instance
         /// </summary>
-        public PlatformID Platform
-        {
-            get { return _platform; }
-        }
+        public PlatformID Platform { get; }
```

Comment:
Switching these three properties to be non-backed is just a refactor that made it easier for me to see where and how these were used.